system/Config/Factories.php

```diff
@@ -290,6 +384,8 @@ public static function reset(?string $component = null)
      *
      * @param string $component Lowercase, plural component name
      * @param string $name      The name of the instance
+     *
+     * @internal For testing only
```

Comment:
Please also add `@testTag` for future language extension support. 

https://github.com/DaveLiddament/php-language-extensions
```suggestion
     * @internal For testing only
     * @testTag
```

Reply:
Added.